radicle-httpd/src/api/v1/projects.rs

```diff
@@ -542,10 +542,30 @@ async fn patch_handler(
 
 #[cfg(test)]
 mod routes {
+    use axum::body::Body;
     use axum::http::StatusCode;
     use serde_json::json;
-
-    use crate::api::test::{self, get, HEAD, HEAD_1};
+    use time::OffsetDateTime;
+
+    use crate::api::test::{self, get, patch, post, HEAD, HEAD_1};
+    use crate::api::{auth, Context};
+
+    /// Adds an authorized session to the Context::sessions HashMap.
+    async fn create_session(ctx: Context) {
```

Comment:
isn't this `create_session` fn better suited to be inside `radicle-httpd/src/api/test.rs`?

Reply:
66674e4